src/plugins/augeas/README.md

```diff
@@ -8,73 +8,70 @@
 - infos/status = maintained unittest libc configurable
 - infos/description = reading and writing configurations via libaugeas
 
-## INTRODUCTION ##
+## Introduction ##
 
 This is a plugin for reading and writing configuration files with help from Augeas.
 The plugin should be able to read all configuration files for which an Augeas lens exists.
 However, not all stock lenses of Augeas have been tested yet.
 A detailed description of the lens language and a tutorial on how to write new lenses
 can be found at http://augeas.net/
 
-## DEPENDENCIES ##
+## Dependencies ##
 
 - `libaugeas-dev`: You need version 0.16 or higher
 
-## INSTALLATION ##
+## Installation ##
 
 If you have installed Augeas manually, it may be necessary to update the ld configuration. This is especially
 true if an older version of Augeas is installed also. Such a situation may lead to an error similar to this:
 
-/usr/lib/libaugeas.so.0: version `AUGEAS_0.16.0` not found (required by kdb)
+    /usr/lib/libaugeas.so.0: version `AUGEAS_0.16.0` not found (required by kdb)
 
-This is because ld tries to link /usr/lib/libaugeas.so.0 which is an older version of Augeas. Simply add
+This is because `ld` tries to link `/usr/lib/libaugeas.so.0` which is an older version of Augeas. Simply add
 the path to the newer library to your ld search paths (consult your system documentation on how to do this).
 
-
-## MOUNTING AND CONFIGURATION ##
+## Mounting and Configuration ##
 
 In order to mount the hosts file with the augeas plugin, issue the
 following command:
 
-	kdb mount /etc/hosts system/hosts augeas lens=Hosts.lns
+    kdb mount /etc/hosts system/hosts augeas lens=Hosts.lns
 
 The value of the plugin configuration option "lens" should be the
 module name of the lens (Hosts in the example) with a '.lns' suffix.
 Depending on your distribution and kind of installation, lenses can
-be found at `/usr/share/augeas/lenses/dist`,
+be found at `/usr/share/augeas/lenses/dist`, 
 `/usr/local/share/augeas/lenses/dist`, or something similar.
 The lens module name is equal to the filename without extension in pascal notation.
 For example, the lens `/usr/share/augeas/lenses/dist/hosts.aug` contains the module Hosts.
 
 Note that, without configuring the plugin to use a lens, the plugin
 will print an error message on the first usage:
 
-	kdb ls system/hosts
-	The command ls terminated unsuccessfully with the info: Error (#85) occurred!
-	Description: an Augeas error occurred
-	Ingroup: plugin
-	Module: storage
-	At: /path/augeas.c:166
-	Reason: Lens not found
+    kdb ls system/hosts
```

Comment:
> Formatters

Thanks for looking them up!

> Regression: Make a decision or script that checks format?

Yes, please document what you found out. Especially remark sounds very promising. We should try to run `remark` like we do with clang-format. It also seems to support more or less useful things like "Auto-link references like in GitHub issues" or "Adjust the gap between headings".

Btw. do you know something about the quality of the man pages remark generates? Maybe we can use it instead of ronn (at least in long term)?

Did you look how many warnings are issued by remark-lint?